addons/mail/models/mail_role.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class MailRole(models.Model):
+    _name = "mail.role"
+    _description = "Role in Mail"
```

Comment:
This description is not really useful, doesn't really make sense.
It should explain what is a role instead.